khan/more-sql-3.sql

```diff
@@ -0,0 +1,57 @@
+/* Challenge: The wordiest author
+
+STEP 1: We've created a database of a few popular authors and their books, with word counts for each book. In this first step, select all the authors who have written more than 1 million words, using GROUP BY and HAVING. Your results table should include the 'author' and their total word count as a 'total_words' column.
+
+STEP 2: Now select all the authors that write more than an average of 150,000 words per book. Your results table should include the 'author' and average words as an 'avg_words' column.
+*/
+
+CREATE TABLE books (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    author TEXT,
+    title TEXT,
+    words INTEGER);
+
+INSERT INTO books (author, title, words)
```

Comment:
instead of doing separate insert sql statements for every row, you can do multiple for one

``` INSERT INTO employees
(employee_id, last_name, first_name)
VALUES
(10, 'Anderson', 'Sarah'),
(11, 'Johnson', 'Dale'); ```